lib/chat_web/live/main_live/page/onliners_presence.ex

```diff
@@ -0,0 +1,47 @@
+defmodule ChatWeb.MainLive.Page.OnlinersPresence do
```

Comment:
It is more like `ChatWeb.OnlinersPresence`

Reply:
In future it could be used with GraphQL API.
Which might not have socket with usual assigns.
So this is task of caller to provide keys and/or instances

Reply:
1. Presence is not coupled LiveView or socket. My idea is to keep it in web part and uncouple from socket. So the values will  be extracted from socket in page it is called.
2. Page represents UI page. But this file does not. It should not be here.

Reply:
I understand, but the keys gathering code etc. still needs to be in one place otherwise it will get duplicated in modules `ChatWeb.MainLive.Index`, `ChatWeb.MainLive.Page.Lobby` and `ChatWeb.MainLive.Page.ImportOwnKeyRing` so I've moved the code coupled with `%Phoenix.LiveView.Socket{}` to the module `ChatWeb.MainLive.Page.Shared`, and the interface-agnostic code to `ChatWeb.OnlinersPresence`.